.github/workflows/build-and-push-multiarch-image.yml

```diff
@@ -187,6 +187,7 @@ jobs:
           labels: |
             org.opencontainers.image.vendor=GDS
           tags: |
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```

Comment:
The format looks redundant here, I think?

Reply:
Just going by what the documentation says.

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions#format

Reply:
`format('refs/heads/{0}', 'main')` is always gonna evaluate to `'refs/heads/main'`.